stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -475,6 +475,7 @@ function find_start_brace(s::AbstractString; c_start='(', c_end=')')
     return (startind:lastindex(s), method_name_end)
 end
 
+const REPLCacheOwner = :REPL # For now symbol (or even type is fine)
```

Comment:
@aviatesk I think we can now remove `REPLInterpreterCache`, but it also looks like it previously wasn't listening to invalidations?